src/common/utxobased/info/ecash.ts

```diff
@@ -0,0 +1,83 @@
+import { Psbt } from 'altcoin-js'
+import { asCodec, asString } from 'cleaners'
+import { EdgeCurrencyInfo } from 'edge-core-js/types'
+
+import { CoinInfo, EngineInfo, PluginInfo } from '../../plugin/types'
+import { maximumFeeRateCalculator } from '../../plugin/util/maximumFeeRateCalculator'
+import { utxoCustomFeeTemplate, utxoMemoOptions } from './commonInfo'
+
+const currencyInfo: EdgeCurrencyInfo = {
+  currencyCode: 'XEC',
+  customFeeTemplate: utxoCustomFeeTemplate,
+  displayName: 'eCash',
+  memoOptions: utxoMemoOptions,
+  pluginId: 'ecash',
+  walletType: 'wallet:ecash',
+
+  // Explorers:
+  blockExplorer: 'https://blockchair.com/ecash/block/%s',
+  addressExplorer: 'https://explorer.e.cash/address/ecash:%s',
+  transactionExplorer: 'https://explorer.e.cash/tx/%s',
+
+  denominations: [
+    { name: 'XEC', multiplier: '100', symbol: 'e' },
+    { name: 'sats', multiplier: '1', symbol: 's' }
+  ],
+  defaultSettings: {
+    customFeeSettings: ['satPerByte'],
+    blockbookServers: ['wss://blockbook.fabien.cash/websocket'],
+    enableCustomServers: false
+  },
+  metaTokens: [],
+  chainDisplayName: '',
+  assetDisplayName: ''
+}
+
+const engineInfo: EngineInfo = {
+  formats: ['bip44'],
```

Comment:
Missing `bipb32` format support. I'm assuming eCash should also support this address format. See our [key formats documentation](https://github.com/EdgeApp/edge-core-js/blob/master/docs/key-formats.md).